postgres-derive/src/composites.rs

```diff
@@ -14,7 +14,13 @@ impl Field {
 
         let ident = raw.ident.as_ref().unwrap().clone();
         Ok(Field {
-            name: overrides.name.unwrap_or_else(|| ident.to_string()),
+            name: overrides.name.unwrap_or_else(|| {
```

Comment:
@dtolnay am I right in thinking that there's nothing that uses IdentFragment to create a string?

Reply:
Not that I know of. IdentFragment isn't intended for anything other than using in format_ident.